quartz/components/ContentMeta.tsx

```diff
@@ -38,7 +41,11 @@ export default ((opts?: Partial<ContentMetaOptions>) => {
         segments.push(displayedTime)
       }
 
-      return <p class={classNames(displayClass, "content-meta")}>{segments.join(", ")}</p>
+      const segmentsElements = segments.map((segment, index) => (
+        <span show-comma={options.showComma}>{segment}</span>
+      ))
+
+      return <p class={classNames(displayClass, "content-meta")}>{segmentsElements}</p>
```

Comment:
let's move the `show-comma` attribute to the `<p>` tag and adjust that in the css